packages/session-replay-browser/test/session-replay-factory.test.ts

```diff
@@ -1,11 +1,19 @@
+import * as RemoteConfigFetch from '@amplitude/analytics-remote-config';
 import { SessionReplay } from '../src/session-replay';
 import { getLogConfig } from '../src/session-replay-factory';
 
 describe('session replay factory', () => {
+  let getRemoteConfigMock: jest.Mock;
+  beforeEach(() => {
+    getRemoteConfigMock = jest.fn();
+    jest.spyOn(RemoteConfigFetch, 'createRemoteConfigFetch').mockResolvedValue({
+      getRemoteConfig: getRemoteConfigMock,
+    });
+  });
   describe('getLogConfig', () => {
     test('return the log config if config defined on session replay', async () => {
       const sessionReplay = new SessionReplay();
-      await sessionReplay.init('apiKey', {});
+      sessionReplay.init('apiKey', {});
```

Comment:
Any reason why we wouldn't want to keep the await here such as?
```suggestion
      await sessionReplay.init('apiKey', {}).promise
```

Reply:
Yep can do that, that's a better idea, luckily for this test I don't think either will matter!